bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_manager.cpp

```diff
@@ -74,7 +74,10 @@ repo::core::model::RepoScene* ModelImportManager::ImportFromFile(
 				error = REPOERR_NO_MESHES;
 			}
 			else {
-				if (config.shouldRotateModel() || modelConvertor->requireReorientation()) {
+				std::string ext = repo::lib::getExtension(file);
+				repo::lib::toLower(ext);
+				bool isFbxFile = ext == ".fbx";
+				if ((config.shouldRotateModel() && !isFbxFile) || modelConvertor->requireReorientation()) {
```

Comment:
instead of adding a bespoke logic here, what you actually want to do is remove the logic here https://github.com/3drepo/3drepobouncer/blob/master/client/src/functions.cpp#L482

Reply:
We have removed the code for FBX file check as suggested by you, But we need to handle rotation for other 3D files hence added checking for FBX file at https://github.com/3drepo/3drepobouncer/blob/ISSUE_515/client/src/functions.cpp#L482